### PR TITLE
Landing page

### PR DESCRIPTION
**Pull request by GabrielMarquez0852**
- Added a new component to stack for activity intros
- Created component for activity information
- allowed the activities navigation to pass specific activity parameters to the activity intro component. This is done using route

### CODE REVIEW

File: src/components/IntroActivity.js [hunk 1]
```diff
@@ -0,0 +1,75 @@
+import React, { useState, useEffect } from "react";
+import { View, Text, Image, ScrollView, TouchableOpacity } from "react-native";
+import styles from "../stylesheets/boxBreathingStyles";
```
Comment (latrujil913) [inline src/components/IntroActivity.js:3]: make introAct styles sheet

**Review by Zoolyn — COMMENTED** [review]
Looks good to me!

**Review by jackzhen1996 — APPROVED** [review]
Looks good, did the intro component work in box breathing?

**Review by latrujil913 — APPROVED** [review]
Looks good

**Comment by GabrielMarquez0852** [comment]
@jackzhen1996 
I removed the one you had in your branch but I'm going to add the route to box breathing so that it will navigate to the component I built with the appropriate parameters.

**Pull request merged**